src/utils/enum/ContactAttemptEnum.ts

```diff
@@ -46,24 +53,15 @@ const fieldContactAttemptEnum = {
   PERMANENTLY_UNAVAILABLE: contactAttemptEnum.PERMANENTLY_UNAVAILABLE,
 } as const;
 
-export const getContactAttemptByConfiguration = (configuration: string, medium: string) => {
+export const getContactAttemptByConfiguration = (medium: ContactAttemptMedium) => {
```

Comment:
question: data structure is static now, should `getContactAttempt`**`ByConfiguration`** still be a function doing a switch (and with no configuration args 🙃 )?
could ContactAttemptEnums be exposed as :
{ 'type' : ContactAttemptMedium,
 'enum' : phoneContactAttemptEnum | fieldContactAttemptEnum | mailContactAttemptEnum | {}
 }